source/includes/fact-mapping-options-rule-filter.rst

```diff
@@ -0,0 +1,10 @@
+Allows you to explicitly include rows where an expression returns 
+``true```. This differs from :ref:`Table Filters <rm-table-filters>`, 
```

Comment:
Extra ``` in true:

```suggestion
``true``. This differs from :ref:`Table Filters <rm-table-filters>`, 
```